### ISSUE TITLE
5.62 Nav/NavBar typing regression

### ISSUE THREAD

**Issue by mxschmitt** [rsuite/rsuite#3821]
### What version of rsuite are you using?

5.62

### What version of React are you using?

18.3.1

### What version of TypeScript are you using (if any)?

5

### What browser are you using?

N/A

### Describe the Bug

After https://github.com/rsuite/rsuite/pull/3813 components like `Navbar` don't get recognised as JSX components anymore, because the user's TS cannot resolve `../@types/common`.

### Expected Behavior

Works.

### To Reproduce

https://github.com/mxschmitt/vite-rsuite-v1.62-regression

`npm i && npm run build`.

Note: 1.62: bad
Note: 1.61: good

**Comment by simonguo** [comment]
Hi @mxschmitt 

I have discovered this bug, which is caused by not converting the paths in the .d.ts file when building the release. You can temporarily solve it by adding a `paths` in `tsconfig.json`.

https://github.com/mxschmitt/vite-rsuite-v1.62-regression/blob/main/tsconfig.json

```tsx
{
  "compilerOptions": {
    ...
    "paths": {
      "@/internals/*": ["./node_modules/rsuite/esm/internals/*"],
    }
  },
...
}
```

I've fixed it(https://github.com/rsuite/rsuite/pull/3820) and will release a new version as soon as I can verify it again

**Comment by simonguo** [comment]
Fixed in 5.62.1

**Comment by mxschmitt** [comment]
I can confirm its fixed in 5.62.1.
